taskweaver/planner/planner.py

```diff
@@ -211,7 +214,7 @@ def check_post_validity(post: Post):
                 f"{self.prompt_data['planner_response_schema']}"
                 "Please try to regenerate the Planner output.",
                 send_to="Planner",
-                send_from="Planner",
+                send_from="User",
```

Comment:
Need to consider the need to find all the posts where the Planner is trying to correct itself. If changed to User, then it becomes hard to track.

Reply:
revised to Planner->Planner